packages/example-next/components/ConnectWithSelect.tsx

```diff
@@ -53,7 +53,16 @@ export function ConnectWithSelect({
 }) {
   const isNetwork = connector instanceof Network
   const displayDefault = !isNetwork
-  const chainIds = (isNetwork ? Object.keys(URLS) : Object.keys(CHAINS)).map((chainId) => Number(chainId))
+  const chainIds = (() => {
+    switch (true) {
```

Comment:
TIL this is the best way to switch on instanceof https://log.havrlant.cz/typescript-switch-instanceof/

Reply:
Thanks!

Reply:
sorry - your original code is better! i was just doing research but the constructor has an edge case it seems around subclasses so i think switch true is the right choice

Reply:
TIL. I didn't know you could do _either_ of these!